core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -1380,13 +1476,13 @@ public void setConnectTimeoutMillis(int connectTimeoutMillis) {
      *
      * @param listener a listener for chain download events, may not be null
      */
-    public void startBlockChainDownload(PeerDataEventListener listener) {
+    public void startBlockChainDownload(ChainDownloadStartedEventListener listener) {
```

Comment:
I fear some incompatibility here. Previously I passed my PeerDataEventListener and listened onBlocksDownloaded(). This doesn't seem possible any more at this point.

Reminds me we should get rid of this notion of "starting a block chain download". Or even the word "download". It's not a single process that starts at a specific time. It's an ongoing process, more like a "mode". There used to be a stone old ticket open for this.